### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by turan18**
Signed-off-by: Yasin Turan <turyasin@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# 1.0.0
+* Removes unnecessary requirements for loading snapshots. (#433)
+* Adds snapshot loading capabilities. (#414)
+* Adds ability to change engine IO type. (#411) 
+* Adds ability to get Firecracker Version. (#410)
+* Deprecates pkg/errors. (#404)
```
Comment (austinvazquez) [inline CHANGELOG.md:6]: I think we can exclude this change from the changelog. A user probably wouldn't care.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+# 1.0.0
+* Removes unnecessary requirements for loading snapshots. (#433)
```
Comment (austinvazquez) [inline CHANGELOG.md:2]: This was a bug fix for a feature added in this release so can probably exclude it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+* Adds snapshot loading capabilities. (#414)
+* Adds ability to change engine IO type. (#411) 
+* Adds ability to get Firecracker Version. (#410)
```
Comment (austinvazquez) [inline CHANGELOG.md:5]:
```suggestion
* Added support for load snapshot
* Added support for set drive IO engine type
* Added support for get Firecracker version
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
+# 1.0.0
+* Added support for load snapshot. (#414)
+* Added support for set drive IO engine type. (#411) 
+* Added support for get Firecracker version. (#410)
+* Adds a DriveOpt func for configuring CacheType. (#401)
+* Adds GetExportVMConfig to read the /vm/config endpoint. (#400)
+* Adds ability to allow passing cgroup-version to jailer. (#399)
+* Adds support for the MMDS configuration. (#290)
+* Adds PauseVM, ResumeVM, and CreateSnapshot capabilities. (#278)
+* Fixes InitrdPath usage. (#295)
```
Comment (austinvazquez) [inline CHANGELOG.md:10]: A few comments I had:

1. We should settle on a single tense. I prefer past e.g. "Added" over "Adds" Thoughts?
2. Should we group like features? e.g. Added support for create and load snapshot (#278, #414)

Comment (turan18) [reply CHANGELOG.md:10]: 1.) I do think that makes sense. I just wanted to be more in-line with previous change logs in older releases as they also use present tense. 

2.) I agree. That is more comprehensive.

Comment (turan18) [reply CHANGELOG.md:10]: Is the consensus that we use past tense for the changelog?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,13 @@
+# 1.0.0
+* Added support for set drive IO engine type. (#411) 
+* Added support for get Firecracker version. (#410)
+* Added a DriveOpt func for configuring CacheType. (#401)
+* Added GetExportVMConfig to read the /vm/config endpoint. (#400)
+* Added ability to allow passing cgroup-version to jailer. (#399)
+* Added support for the MMDS configuration. (#290)
+* Added PauseVM, ResumeVM, CreateSnapshot and LoadSnapshot capabilities. (#278)
```
Comment (austinvazquez) [inline CHANGELOG.md:8]: Wouldn't we need the second link because load snapshot came in a separate PR?

**Review by austinvazquez — APPROVED** [review]

**Review by ginglis13 — APPROVED** [review]

**Review by austinvazquez — APPROVED** [review]

**Pull request merged**